src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -20,9 +20,12 @@ data class PremisesEntity(
   val id: UUID,
   val name: String,
   val apCode: String,
+  val address: String,
```

Comment:
In our [wireframes](https://miro.com/app/board/uXjVPQkTpYI=/) we have "address line 1" and "address line 2", I wonder if we should call this field `address_line_1` just in case we want to add that second field in the future. I think this is what we decided on our trello card.